runtime/tests/shared/SCStoreTransactionTests.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2001, 2014 IBM Corp. and others
+ * Copyright (c) 2001, 2020 IBM Corp. and others
```

Comment:
This change is unnecessary.

Reply:
Copyright Check would fail instead, right?